plugins/modules/fusion_se.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2022, Simon Dodsley (simon@purestorage.com)
```

Comment:
nit: Please change the date and use your name

Reply:
Fixed